edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/EdgeDeploymentController.cs

```diff
@@ -82,6 +82,13 @@ public async Task<EdgeDeploymentStatus> DeployModulesAsync(ModuleSet desiredModu
                     [KubernetesConstants.K8sEdgeHubNameLabel] = KubeUtils.SanitizeLabelValue(this.resourceName.Hostname)
                 };
 
+                var desiredServiceAccounts = desiredModules.Modules
```

Comment:
So the serviceaccount is needed before creating PVC. Thanks for finding this.

Reply:
It's not required.  We should move this up because the deployment references it.  (We have another PBI for this, I chose to change it now since I was testing agent E2E.)